app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="devise-container">
+  <h2>Log in</h2>
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+<div class="field">
+<%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email", class: "input-field" %>
+</div>
+<div class="field">
+  <%= f.password_field :password, autocomplete: "current-password", placeholder: "Password", class: "input-field"  %>
+</div>
```

Comment:
- [ ] Greate work on styling the login and sign-up pages, kindly make sure to center the elements of the page so they look exactly as the provided design

|Your website|Provided wirefram|
|-|-|
|![Screenshot_20230701_133208](https://github.com/brainskev/Blog-App/assets/25227022/d2242960-370d-4100-a4b7-37a82fd7a0c2)|![](https://user-images.githubusercontent.com/69077061/250266370-35633f09-f9b4-4ac0-8d38-0fc1e0bf7c89.png)|